.gitignore

```
@@ -66,4 +66,5 @@ ___*
 .env/workaround.bat
 .env/installed_conda_pkgs
 .env/installed_pip_pkgs
-.env/installed_spec.json
\ No newline at end of file
+.env/installed_spec.json
+!.pre-commit-config.yaml
```

Github complains about no newline at the end of the file; I do, too. :-) Same for the other .gitignore files.

```suggestion
!.pre-commit-config.yaml

```

---

ok, but do not bother to check the other stuff yet, still in progress